### PR TITLE
Breathing Rotating Button for GSoC

### PR DESCRIPTION
**Pull request by OmiWakode**
A Sample Breathing and rotating button Animation by [https://github.com/OmiWakode](url)
DartPad [https://dartpad.dev/14e84dd83f14c9df47971761463bb474](url)

BreatheRotate

### CODE REVIEW

**Comment by divyanshub024** [comment]
@OmiWakode You need to make changes in the read me. Please go through the contribution guide https://github.com/divyanshub024/awesome-dart-pad/blob/master/Contribution.md

**Comment by OmiWakode** [comment]
> @OmiWakode You need to make changes in the read me. Please go through the contribution guide https://github.com/divyanshub024/awesome-dart-pad/blob/master/Contribution.md

I have updated the readme.md 
Thank you

File: README.md [hunk 1]
```diff
@@ -62,6 +62,10 @@ If you appreciate the content 📖, support projects visibility, give 👍| ⭐|
 
 [<img src="https://user-images.githubusercontent.com/60287642/76142880-bb8fab00-6026-11ea-8522-81fb15f9ff43.gif" width=240>]
 
+ - [Breathing Rotating Animation](https://dartpad.dev/14e84dd83f14c9df47971761463bb474) by [Omi Wakode](https://github.com/OmiWakode)
```
Comment (divyanshub024) [inline README.md]: Please add a date as well.

Comment (divyanshub024) [reply README.md]: Also, you need to add it at the end of the list. Please go through the contribution guide before sending a PR.

**Review by divyanshub024 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (OmiWakode) [reply README.md]: Sorry for the ill formated commit! I've made the changes you asked.

Comment (OmiWakode) [reply README.md]: Sorry for the ill formatted commit. I've made the changes you asked

**Comment by divyanshub024** [comment]
Thanks a lot, @OmiWakode for the awesome sample.

**Comment by OmiWakode** [comment]
> Thanks a lot, @OmiWakode for the awesome sample.

Thanks for the merge! will continue to learn and share new stuff in dartand flutter

**Comment by OmiWakode** [comment]
> Thanks a lot, @OmiWakode for the awesome sample.

Thanks for the merge! will continue to learn and share cool stuff from dart and flutter

**Pull request merged**